Frends.SFTP.DeleteFiles/Frends.SFTP.DeleteFiles/Frends.SFTP.DeleteFiles.cs

```diff
@@ -24,7 +25,7 @@ public static class SFTP
     /// <param name="connection">Connection parameters.</param>
     /// <param name="cancellationToken">Cancellation token given by Frends.</param>
     /// <returns>Object { List [ Object { string Name, string Path, Double SizeInMegaBytes } ] }.</returns>

```

Comment:
Please also add property name `Files`, since you need to refer to that if you want to use the List.